src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefReadonlyReturnTests.cs

```diff
@@ -392,14 +394,16 @@ static void M()
     static ref int Helper()
         => ref (new int[1])[0];
 }");
-            verifier.VerifyIL("C.M()", @"
+            verifier.VerifyIL("C.M()", """
 {
-  // Code size        7 (0x7)
+  // Code size        8 (0x8)
   .maxstack  1
-  IL_0000:  call       ""ref int C.Helper()""
-  IL_0005:  pop
-  IL_0006:  ret
-}");
+  IL_0000:  call       "ref int C.Helper()"
+  IL_0005:  ldind.i4
+  IL_0006:  pop
+  IL_0007:  ret
+}
+""");
         }
```

Comment:
> } [](http://example.com/codeflow?start=7&length=2)

Consider adding a test with two `ref` locals that are unused, and execute the test.
```
ref readonly int x = ref Helper();
ref readonly int y = ref Helper();
_ = x + y;
```